pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -392,11 +392,11 @@ def _minmax_obj_vs_node(self, ntss_tmp, bool_print: bool = False):
 
     def distrib_nn_for_cdf(self, ntss_tmp, bool_print: bool = False):
         """
-        Calcule les deux indicateurs, moyenne et écart-type des distances, nécessaires pour l'utilisation de la CDF de la loi normale.
-        Le calcul de ces indicateurs sont décrit dans `Scoring Message Stream Anomalies in Railway Communication Systems, L.Foulon et al., 2019, ICDMWorkshop <https://ieeexplore.ieee.org/abstract/document/8955558>`_.
+        Calculates the two indicators, average and standard deviation of the distances, necessary for the use of the CDF of the normal law.
```

Comment:
Calcule -> Compute
of the normal law -> normal distribution

Reply:
```suggestion
        Calculates the two indicators, average and standard deviation of the distances, necessary for the use of the CDF of the normal distribution.
```

Reply:
```suggestion
        Computes the two indicators, average and standard deviation of the distances, necessary for the use of the CDF of the normal law.
```